core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -430,6 +435,21 @@ public String serviceLoggerPrefix() {
         return accessLogWriter;
     }
 
+    /**
+     * Returns whether PROXY protocol is enabled.
+     */
+    public boolean acceptProxyProtocol() {
+        return acceptProxyProtocol;
+    }
+
+    /**
+     * Returns the maximum size of additional data (TLV, Tag-Length-Value). It is only used when
+     * {@link #acceptProxyProtocol()} is {@code true}.
+     */
+    public int proxyProtocolMaxTlvSize() {
```

Comment:
Could return `null` or some invalid value (0? -1?) if PROXY protocol is disabled.